orchagent/vxlanorch.cpp

```diff
@@ -1536,28 +1616,44 @@ void VxlanTunnelOrch::deleteTunnelPort(Port &tunnelPort)
     if (!ret) 
     {
         SWSS_LOG_ERROR("Remove Bridge port failed for remote = %s fdbcount = %d", 
-                      remote_vtep.c_str(), tunnelPort.m_fdb_count);
+                       remote_vtep.c_str(), tunnelPort.m_fdb_count);
         return;
     }
     gPortsOrch->removeTunnel(tunnelPort);
 
     // Remove DIP Tunnel HW 
     vtep_ptr->deleteDynamicDIPTunnel(remote_vtep, TUNNEL_USER_IMR, false);
     SWSS_LOG_NOTICE("diprefcnt for remote %s = %d",
-                     remote_vtep.c_str(), vtep_ptr->getDipTunnelRefCnt(remote_vtep));
-
+                    remote_vtep.c_str(), vtep_ptr->getDipTunnelRefCnt(remote_vtep));
     // Remove SIP Tunnel HW which might be pending on delete
     vtep_ptr->deletePendingSIPTunnel();
 
     return ;
 }
 
-std::string VxlanTunnelOrch::getTunnelPortName(const std::string& remote_vtep)
+std::string VxlanTunnelOrch::getTunnelPortName(const std::string& vtep, bool local)
 {
-    std::string tunnelPortName = "Port_EVPN_" + remote_vtep;
+    
+    std::string tunnelPortName;
+    if (local)
+    {
+        tunnelPortName = "Port_SRC_VTEP_" + vtep;
```

Comment:
These tunnel port prefixes are hard coded in few places (at least "Port_SRC_VTEP_" and maybe "Port_EVPN_" as well).

Is it possible to have it defined in one place (e.g. ```#define``` or ```const```) and then just use it in all places it is needed?
It would be safer like that in case for example in future prefix will be changed to something else and then we will need to change it just once and it will take effect in all places.

Reply:
Added macro as suggested